espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Server.java

```diff
@@ -24,12 +24,12 @@ public Server() throws IOException, DuplicateRouteException {
         router = new Router();
     }
 
-    private String[] getStackTrace(Exception e) {
+    private ArrayList<String> getStackTrace(Exception e) {
```

Comment:
It is recommended to always use abstract/interface types instead of concrete types (`ArrayList<String>` -> `List<String>` )